### PR TITLE
fix: argos-ci diff for Upload and Descriptions

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

修复 https://www.argos-ci.com/ant-design/ant-design/builds/123 里的 Upload 和 Descriptions 组件的 UI 差异点。

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    ---       |
| 🇨🇳 Chinese |   ---        |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/descriptions/style/index.tsx [hunk 1]
```diff
@@ -180,15 +180,15 @@ const genDescriptionStyles: GenerateStyle<DescriptionsToken> = (token: Descripti
           },
         },
       },
-      [`${componentCls}-middle`]: {
+      '&-middle': {
```
Comment (afc163) [inline components/descriptions/style/index.tsx:183]: https://github.com/ant-design/ant-design/pull/34786

cc @Dunqing

Comment (Dunqing) [reply components/descriptions/style/index.tsx:183]: 1

写法看起来没啥区别

之前没写 `&` 好像是因为不生效

Comment (afc163) [reply components/descriptions/style/index.tsx:183]: 有区别，之前的写法渲染出来是：

```css
.ant-descriptions .ant-descriptions-middle {
}
```

Comment (Dunqing) [reply components/descriptions/style/index.tsx:183]: 了解

**Pull request merged**